app/src/androidTest/java/com/github/se/cyrcle/ui/report/AdminScreenTest.kt

```diff
@@ -156,4 +176,65 @@ class AdminScreenTest {
     composeTestRule.onNodeWithTag("ReportCard0").performClick()
     composeTestRule.onNodeWithTag("CheckReportsButton0").performClick()
   }
+
+  @Test
+  fun testAlertDialogAppearsOnMoreOptionsClick() {
+    composeTestRule.setContent {
+      AdminScreen(
+          navigationActions = navigationActions,
+          reportedObjectViewModel = reportedObjectViewModel,
+          parkingViewModel = parkingViewModel,
+          reviewViewModel = reviewViewModel)
+    }
+    composeTestRule.onNodeWithTag("ReportCard0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNodeWithTag("MoreOptionsButton0").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNode(isDialog()).assertIsDisplayed()
+    composeTestRule.onNodeWithText("Details for this Reported Object:").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Loading parking details...").assertIsDisplayed()
+    composeTestRule.onNodeWithText("Go To Parking").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").assertIsDisplayed().assertHasClickAction()
+    composeTestRule.onNodeWithText("Return").performClick()
+    composeTestRule.waitForIdle()
+    composeTestRule.onNode(isDialog()).assertIsNotDisplayed()
+  }
+
+  @Test
+  fun testReportDetailsContentForParkingType() {
+    composeTestRule.setContent {
+      ReportDetailsContent(
+          objectType = ReportedObjectType.PARKING,
+          objectUID = parking1.uid,
+          userUID = parking1.owner,
+          reviewViewModel = reviewViewModel,
+          parkingViewModel = parkingViewModel)
+    }
+    composeTestRule
+        .onNodeWithTag("BulletPointParkingUID")
+        .assertIsDisplayed()
+        .onChildren()
+        .assertAny(hasText("Parking UID: Test_spot_1"))
+
+    composeTestRule
+        .onNodeWithTag("BulletPointParkingOwner")
+        .assertIsDisplayed()
+        .onChildren()
+        .assertAny(hasText("Owner: user1"))
+  }
+
+  @Test
+  fun testReportDetailsContentForImageType() {
```

Comment:
I have trouble understanding the value inside this test. It seems redondant with respect to the previous test. Could you explain the logic behind ?

Reply:
Covering more lines, honestly

Reply:
Maybe there is a specific property of the image that you could test that you don't in the parking ?

Reply:
Maybe the ImageID then, works for you?

Reply:
Works perfectly fine the test will be more relevant this way thanks !